client/my-sites/upgrades/map-domain/index.jsx

```diff
@@ -82,7 +82,7 @@ var MapDomain = React.createClass( {
 		upgradesActions.addItem(
 			cartItems.domainRegistration( {
 				productSlug: suggestion.product_slug,
-				domain: suggestion.domain
+				domain: suggestion.domain_name
```

Comment:
This was causing an error when trying to add a domain registration from the domain mapping view - the proper key is `domain_name`, not `domain`.